### ISSUE TITLE
liner regression

### ISSUE THREAD

**Issue by gourinandakm04** [gourinandakm04/GOURINANDA-K.-M#2]
import numpy as nm 
import matplotlib.pyplot as mtp 
import pandas as pd 
data_set= pd.read_csv('/content/drive/MyDrive/sem3 dataset/testing,linear regression.csv 
x= data_set.iloc[:, :-1].values 
y= data_set.iloc[:, 1].values 
# Splitting the dataset into training and test set. 
from sklearn.model_selection import train_test_split 
x_train, x_test, y_train, y_test= train_test_split(x, y, test_size= 1/3, random_state=0) 
#Fitting the Simple Linear Regression model to the training dataset from sklearn.linear_model import LinearRegression 
regressor= LinearRegression() 
regressor.fit(x_train, y_train)
#Prediction of Test and Training set result 
y_pred= regressor.predict(x_test) 
x_pred= regressor.predict(x_train) 
df_preds = pd.DataFrame({'Actual': y_test.squeeze(), 'Predicted': y_pred.squeeze()}) print(df_preds) 
#visualizing the Test set results 
mtp.scatter(x_test, y_test, color="blue") 
mtp.plot(x_train, x_pred, color="red") 
mtp.title("Salary vs Experience (Test Dataset)") 
mtp.xlabel("Years of Experience") 
mtp.ylabel("Salary(In Rupees)") 
mtp.show() 
regressor.intercept_ 
regressor.coef_ 
regressor.predict([[1.5]])
